api/managementpb/actions.proto

```diff
@@ -17,6 +17,7 @@ enum ActionType {
   POSTGRESQL_SHOW_CREATE_TABLE = 5;
   POSTGRESQL_SHOW_INDEX = 6;
   MONGODB_EXPLAIN = 7;
+  PT_SUMMARY = 9;
```

Comment:
Why 9 and not 8? :)

Reply:
My fault, I added same number like for PT sumary params before.